src/lib/containers/widgets/query-filter/FacetChip.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react'
+import { unCamelCase } from '../../../utils/functions/unCamelCase'
+import { FacetColumnResult } from '../../../utils/synapseTypes'
+import IconSvg from '../../IconSvg'
+import { useQueryVisualizationContext } from '../../QueryVisualizationWrapper'
+
+export type FacetChipProps = {
+  facet: FacetColumnResult
+  facetFilter: string[]
+  setFacetFilter: Dispatch<SetStateAction<string[]>>
```

Comment:
Let's keep this simple so all of the click-to-show/hide logic is all in one place.

Replace these props with 
```
isChecked: boolean
onClick: () => void
```

These can be set in QueryFilter.tsx (where all of the other complex logic is happening) and then be passed down.